docs/src/modeling/expressions/functions/README.md

```diff
@@ -57,4 +59,15 @@ gespeichert wird.
 const tFileCoos = JSON.parse(execution.getVariable('sachakten'));
 const tFirstCoo = tFileCoos[0];
 execution.setVariable('firstCoo', tFirstCoo);
+```
+
+Um zwei Listen zusammenzuführen und sie wiederum an eine Prozessvariable zu übergeben, muss die zusammengeführte Liste 
+mit `S()` serialisiert werden. Folgendes Beispiel zeigt die Zusammenführung von zwei Listen.
+
+```
+const list_1 = JSON.parse(execution.getVariable('list_1'));
+const list_2 = JSON.parse(execution.getVariable('list_2'));
```

Comment:
Was ist den der Ausgangstyp der Liste? Sind das Strings mit JSON? 
Eigentlich sollte so etwas eher als JSON direkt gespeichert werden, damit man nicht an die 2k String-Grenze kommt.

Reply:
Es ist der Output des DMS-Input-Felds: z. B. ["COO.2150.9151.1.1225735", "COO.2150.9151.1.1225738"]

Reply:
Nur damit wir vom gleichen reden:

Mir ging es um die Typen, die Camunda supportet für Variablen. Man kann natürlich auch ein valides JSON in einem String-Type speichern, aber das ist hoffentlich hier nicht der Fall

![image](https://github.com/it-at-m/digiwf-core/assets/10849776/324ac61c-bd89-4236-bc79-512c39d75b7c)

Reply:
Das sollte Json Type sein. Oder ich habe es komplett falsch verstanden...

![grafik](https://github.com/it-at-m/digiwf-core/assets/18287514/c92ef1cf-0192-404a-bdf4-ef5b744dc3ce)